PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -1,3 +1,70 @@
+$GraphEndpoints = @{
+    "Get-MgBetaIdentityConditionalAccessPolicy" = "https://graph.microsoft.com/beta/identity/conditionalAccess/policies"
+    "Get-MgBetaSubscribedSku" = "https://graph.microsoft.com/beta/subscribedSkus"
+    "Get-MgBetaDirectoryRole" = "https://graph.microsoft.com/beta/directoryRoles"
+    "Get-MgBetaDirectoryRoleMembers" = "https://graph.microsoft.com/beta/directoryRoles/{0}/members"
+    "Get-MgBetaRoleManagementDirectoryRoleEligibilityScheduleInstance" = "https://graph.microsoft.com/beta/roleManagement/directory/roleEligibilityScheduleInstances"
+    "Get-MgBetaRoleManagementDirectoryRoleAssignmentScheduleInstance" = "https://graph.microsoft.com/beta/roleManagement/directory/roleAssignmentScheduleInstances"
+    "Get-MgBetaIdentityGovernancePrivilegedAccessGroupEligibilityScheduleInstance" = "https://graph.microsoft.com/beta/identityGovernance/privilegedAccess/group/eligibilityScheduleInstances"
+    "Get-MgBetaPrivilegedAccessResource" = "https://graph.microsoft.com/beta/privilegedAccess/aadGroups/resources"
+    "Get-MgBetaPolicyAuthorizationPolicy" = "https://graph.microsoft.com/beta/policies/authorizationPolicy"
+    "Get-MgBetaDirectorySetting" = "https://graph.microsoft.com/beta/settings"
+    "Get-MgBetaPolicyAuthenticationMethodPolicy" = "https://graph.microsoft.com/beta/policies/authenticationMethodsPolicy"
+    "Get-MgBetaDomain" = "https://graph.microsoft.com/beta/domains"
+    "Get-MgBetaOrganization" = "https://graph.microsoft.com/beta/organization"
+    "Get-MgBetaUser" = "https://graph.microsoft.com/beta/users"
```

Comment:
We seem to have mapped some APIs (Like this one) that we are not invoking via graph directly but omitted others (such as Get-MgBetaGroup) too.  

Lets limit this to APIs that we actually invoke via graph directly.  Fine to do a test of more if you want to but we should have a method.

Reply:
I whittled it down to the four that we are currently using. The rest can live on in the git history until we need them